src/playground/sample-data-ingestion/generator_unit_test.py

```diff
@@ -0,0 +1,227 @@
+# Test functions
+import json
+from operator import ge
+from types import NoneType
+import generator
+import pytest
+from datetime import date
+
+# Testing stuff
+def generateCommerceData():
```

Comment:
same as above, please follow name conventions https://peps.python.org/pep-0008/#function-and-variable-names